benchmarks/profiler/inject_disagg_config.py

```diff
@@ -162,7 +162,7 @@ def main():
 
     print("\n✅ Disagg config injection completed!")
     print(f"📁 Config available at: {args.target_path}")
-    print(f"🔧 Set DGD_CONFIG_FILE={args.target_path} in your profiler job")
+    print(f"🔧 Set DGD_CONFIG_FILE=/workspace/{args.target_path} in your profiler job")
```

Comment:
let's avoid hardcoding this, maybe using a `workspace_root` arg that has a default of `/workspace`
```suggestion
    print(f"🔧 Set DGD_CONFIG_FILE={args.workspace_root}/{args.target_path} in your profiler job")
```